website/docs/docs/on-premises/releases/1.29.0.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 1.29.0
+---
+
+## Release date
+
+31 January 2024
+
+## Features
+
+* Add verification modes to the GET /integration endpoint: replay or comparison along with other integration details
+* Update the time for a successful can-i-deploy result in bi-directional contracts by reducing the number of historical branches to generate pact/OAS comparisons for
```

Comment:
This one was fine as it was. Can you put it back please?

Reply:
The instructions I gave weren't meant to imply that everything had to start with "Add/update". I was trying to convey that the start of the sentence should be an action in the present tense (Add/update/improve/remove/reduce) rather than a past tense ("added/updated/improved/...") or a statement of fact "The endpoint X returns a Y".

Reply:
Oh I thought we can put `Add` to the front if it is possible